psalm.xml

```diff
@@ -1,10 +1,11 @@
 <?xml version="1.0"?>
 <psalm
-    totallyTyped="true"
-    resolveFromConfigFile="true"
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xmlns="https://getpsalm.org/schema/config"
     xsi:schemaLocation="https://getpsalm.org/schema/config vendor/vimeo/psalm/config.xsd"
+    restrictReturnTypes="true"
+    findUnusedPsalmSuppress="true"
+    totallyTyped="true"
```

Comment:
Psalm docs say this is deprecated, might be dropped as level is max already

https://psalm.dev/docs/running_psalm/configuration/#totallytyped

Reply:
With`totallyTyped="true"` the default level is `1`, but without `totallyTyped` it would be `2`: https://github.com/vimeo/psalm/pull/5340/files
So I think we should replace `totallyTyped="true"` with `errorLevel="1"`.

Reply:
Agree, it's a bit out of scope but has been touched already so 🤷🏾

Reply:
Yeah, I generally remove `errorLevel` wherever I go: it's just confusing.